### PR TITLE
[Feature] support ’show tablet from xx order by xxx‘ for lake table

### PR DESCRIPTION
**Pull request by abc982627271**
## What type of PR is this：
- [ ] BugFix
- [x] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [x] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

### CODE REVIEW

**Comment by abc982627271** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 8 / 14 (57.14%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/proc/LakeTabletsProcNode.java|1|6|16.67%|[42, 43, 44, 46, 48]|
|:large_blue_circle:|com/starrocks/sql/analyzer/ShowTabletStmtAnalyzer.java|7|8|87.50%|[97]|

File: fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ShowTabletStmtAnalyzer.java [hunk 1]
```diff
@@ -81,8 +85,19 @@ public Void visitShowTabletStatement(ShowTabletStmt statement, ConnectContext co
                     }
                     SlotRef slotRef = (SlotRef) orderByElement.getExpr();
                     int index = 0;
+                    Database db = GlobalStateMgr.getCurrentState().getDb(dbName);
+                    String tableName = statement.getTableName();
+                    Table table = db.getTable(tableName);
+                    if (table == null) {
+                        throw new SemanticException("Table %s is not found", tableName);
+                    }
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ShowTabletStmtAnalyzer.java:93]: 1. getting db and table every column is a little expensive.
2. need db lock.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 11 / 17 (64.71%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/proc/LakeTabletsProcNode.java|1|6|16.67%|[42, 43, 44, 46, 48]|
|:large_blue_circle:|com/starrocks/sql/analyzer/ShowTabletStmtAnalyzer.java|10|11|90.91%|[103]|

**Comment by abc982627271** [comment]
run starrocks_fe_unittest

**Comment by abc982627271** [comment]
run starrocks_fe_unittest

**Review by wyb — APPROVED** [review]

**Review by sduzh — APPROVED** [review]

**Comment by dulong41** [comment]
run starrocks_admit_test

**Pull request merged**